src/components/fields/ArrayField.js

```diff
@@ -439,6 +447,7 @@ if (process.env.NODE_ENV !== "production") {
     idSchema: PropTypes.object,
     errorSchema: PropTypes.object,
     onChange: PropTypes.func.isRequired,
+    onBlur: PropTypes.func.isRequired,
```

Comment:
I think I can just remove the isRequired part here. There's no actual use of onBlur in this component.